card/src/main/java/com/adyen/checkout/card/internal/util/DualBrandedCardManager.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 11/8/2022.
+ */
+
+package com.adyen.checkout.card.internal.util
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.card.CardBrand
+import com.adyen.checkout.card.internal.data.model.DetectedCardType
+import com.adyen.checkout.card.internal.ui.model.CardBrandItem
+import com.adyen.checkout.card.internal.ui.model.DualBrandData
+import com.adyen.checkout.core.Environment
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class DualBrandedCardManager(private val environment: Environment) {
```

Comment:
Maybe `DualBrandedCardHandler` would be a more fitting name as it validates and maps the data?